<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-BEK9VA8" failures="2" tests="4" name="Test" time="11.61" errors="0" timestamp="2024-02-10T13:12:57 IST">
  <testcase classname="com.api.tests.UserTest" name="testDeleteUserByName" time="4.059">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at com.api.tests.UserTest.testDeleteUserByName(UserTest.java:75)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
  <testcase classname="com.api.tests.UserTest" name="testUpdateUserByName" time="3.346"/>
  <testcase classname="com.api.tests.UserTest" name="testPostUser" time="1.318"/>
  <testcase classname="com.api.tests.UserTest" name="testGetUserByNae" time="1.302">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at com.api.tests.UserTest.testGetUserByNae(UserTest.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testGetUserByNae -->
</testsuite> <!-- Test -->
